sklearn/preprocessing/_function_transformer.py

```diff
@@ -100,7 +100,7 @@ def _check_input(self, X):
             self._validate = self.validate
 
         if self._validate:
-            return check_array(X, accept_sparse=self.accept_sparse)
+            return check_array(X, accept_sparse=self.accept_sparse,force_all_finite=False)
         return X
```

Comment:
I think, rather, we should be using validate=True.... Just changing FunctionTransformer like this is breaking what we have promised its validation will do.